### PR TITLE
fix: Revert PR for adding required fields of ngo events

### LINKED ISSUE #5991: Ability to add donation support
**Describe the feature**
Our system requires the option of adding NGO events. When an event is categorised as an NGO event, it should have the feature of selling **only free** tickets. It should also have the option where people can donate funds to the fundraising event. 

This is a **parent** issue. It can be divided into sub-parts:
1. Integration of a working donation system
2. Adding NGO events/Open Source events
3. Adding the feature of recurring donations(_Monthly, Yearly etc..._)

**Additional context**
Working on it.

### PR DESCRIPTION
**Pull request by prateekj117**
Fixes #5991 

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream `development` branch.
- [X] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

- [X] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by mrsaicharan1** [comment]
Unnecessary imports are present. please remove.

File: migrations/versions/31a6cf1c9374_.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Create Date: 2019-06-08 00:41:07.974503
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = '31a6cf1c9374'
+down_revision = '89a5b836b406'
```
Comment (mrsaicharan1) [inline migrations/versions/31a6cf1c9374_.py]: Please update the migration file with the latest revision

**Comment by prateekj117** [comment]
@iamareebjamal Please review.

**Comment by iamareebjamal** [comment]
https://github.com/fossasia/open-event-server/pull/6009#discussion_r293096812

**Comment by prateekj117** [comment]
@iamareebjamal Instead of that, I pulled the latest changes of development into this branch and mergeed the migrations heads and checked locally, everything works fine.

**Comment by iamareebjamal** [comment]
Please keep a single migration

**Comment by prateekj117** [comment]
@mrsaicharan1 @uds5501 @shreyanshdwivedi @iamareebjamal Please review.

File: migrations/versions/14e31b3a860b_.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('events', 'is_ngo')
+    op.alter_column('events_version', 'show_remaining_tickets',
+               existing_type=sa.BOOLEAN(),
+               nullable=True,
+               autoincrement=False,
+               existing_server_default=sa.text('false'))
```
Comment (iamareebjamal) [inline migrations/versions/14e31b3a860b_.py]: This should not be here

Comment (prateekj117) [reply migrations/versions/14e31b3a860b_.py]: @iamareebjamal This is a auto-generated migration.

Comment (iamareebjamal) [reply migrations/versions/14e31b3a860b_.py]: Still

File: migrations/versions/14e31b3a860b_.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+# revision identifiers, used by Alembic.
+revision = '14e31b3a860b'
+down_revision = '09c86e38f5e7'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('events', 'is_ngo')
+    op.alter_column('events_version', 'show_remaining_tickets',
```
Comment (mrsaicharan1) [inline migrations/versions/14e31b3a860b_.py]: This seems to be occurring due to some issue with migration commands. Please remove this as we are not making any changes to the remaining tickets column in `events_version`

File: migrations/versions/14e31b3a860b_.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+               autoincrement=False,
+               existing_server_default=sa.text('false'))
+    op.drop_column('events_version', 'is_ngo')
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('events_version', sa.Column('is_ngo', sa.BOOLEAN(), autoincrement=False, nullable=True))
+    op.alter_column('events_version', 'show_remaining_tickets',
```
Comment (mrsaicharan1) [inline migrations/versions/14e31b3a860b_.py]: Same here. Remove this and this is good to go. 
Also, just a pointer, whenever migration files are generated, there is a line which tells you that
`# ### commands auto generated by Alembic - please adjust! ###`

File: migrations/versions/14e31b3a860b_.py (hunk 3 above)
Comment (uds5501) [reply migrations/versions/14e31b3a860b_.py]: @mrsaicharan1 is correct. This automatic script just saw that your events have a `null` `show remaining tickets` column.
That's why this line is being generated. so removing it should be fine.

File: migrations/versions/14e31b3a860b_.py (hunk 4 above)
Comment (uds5501) [reply migrations/versions/14e31b3a860b_.py]: @prateekj117 Same goes with this block

**Comment by prateekj117** [comment]
@mrsaicharan1 @uds5501 @iamareebjamal Made the changes. Please review.

**Pull request merged**